web/views.py

```
@@ -89,6 +92,7 @@
     Storefront,
     StudyGroup,
     TimeSlot,
+    SuccessStory,
```

Ruff (0.8.2)

Redefinition of unused `SuccessStory` from line 35

Remove definition: `SuccessStory`

(F811)